### DISCUSSION TITLE
import geotiff > mesh delaunay > export *.stl

### DISCUSSION THREAD

**Discussion by sugizo** [domlysz/BlenderGIS#904] [Q&A]
**steps**
1. generate geotiff using dem-stitcher
2. import geotiff file via blend gis
![image](https://github.com/domlysz/BlenderGIS/assets/2418295/2d541369-b15a-4759-aa03-e40bd9cda085)

3. use mesh delaunay in blender gis
![image](https://github.com/domlysz/BlenderGIS/assets/2418295/628b0888-5560-4005-b782-cbc0b94b126a)

4. export *.stl from blender
5. result 1 kb (not expected)
![image](https://github.com/domlysz/BlenderGIS/assets/2418295/209dd0f6-6b2b-4fce-bc34-621d6631cb4b)

**question**
how to get correct elevation *.stl file using blender gis ?

best regards

**Comment by domlysz** [comment]
please read the doc : https://github.com/domlysz/BlenderGIS/wiki/Import-DEM-grid

You nead to change the import mode

**Comment by sugizo** [comment]
forgot to tell, 
before ask question in this discussion, 
already tried all of modes, 
but seems nothing work
